### DISCUSSION TITLE
Discord Server?

### DISCUSSION THREAD

**Discussion by NathanDWasTaken** [SeanDaBlack/KelloggBot#37] [Q&A]
I was wondering if there is a discord server for either the KelloggBot or SeanDaBlack's since I'd love to keep up to date on current and future projects like this (and AbBOT)

**Comment by BigweldIndustries** [comment]
Assuming there  isn't one yet but we could make one as a community. Probably should wait for SeanDeBlack's feedback

**Comment by SeanDaBlack** [comment]
There is one we used for the AbBot specifically, so I don't know if we should make a new one or make that one more general purpose.

**Reply by NathanDWasTaken** [reply]
Yes I did see that one, although I was thinking about more of a general server, whether that'd be changing the AbBOT server or making a new one

**Reply by pws1453** [reply]
I'm not sure given the nature of work being done if we want to extend communications beyond this platform...
